pegasus/sites.v3/code.org/views/homepage_sections.haml

```diff
@@ -4,10 +4,11 @@
     - display = block[:active_s] == "hide" ? "none" : "block"
     .block{style: "display: #{display}"}
       - hoc_block = block[:id_s] == "block_hoc" || block[:id_s] == "blockshort_hoc_noten"
-      // if main hero is mc, then hoc block is star wars
+      -# If main hero is mc, then hoc block is star wars.
       - if hoc_block && (page_mode == "feature-minecraft2016" || page_mode == "feature-mc")
         - block[:image_t] = "/shared/images/courses/logo_tall_starwars_footer.jpg"
-      - elsif hoc_block && page_mode == "feature-starwars"
+      -# After mc is launched, show mc hoc block when starwars is main hero.
+        - elsif hoc_block && page_mode == "feature-starwars"
```

Comment:
Yes. So when it's any other page_mode it will use the frozen image from the spreadsheet. And once we launch, we'll activate the starwars page_mode if statement again where we'll overwrite the frozen image with the mc2016 image.